charts/rd-caseworker-ref-api/values.preview.template.yaml

```diff
@@ -14,6 +14,8 @@ java:
     CWRD_DATA_PER_MESSAGE: 50
     LAUNCH_DARKLY_ENV: "preview"
     ENVIRONMENT_NAME: "preview"
+    REFRESH_PAGE_SIZE: 20
+    REFRESH_SORT_COLUMN: caseWorkerId
```

Comment:
We can remove this code as long as the variables are same as "values.yaml"

Reply:
sure